### DISCUSSION TITLE
Feedback - JS

### DISCUSSION THREAD

**Discussion by Speb04** [WillTheDeveloper/StudyPortal#16] [General]
Its a really good idea, I like the thought of bringing everything that the students use into one area. The added functionality of having things like a community, kamban and calendar all together. I think this could add a lot of value to students.

**Comment by WillTheDeveloper** [comment]
Thanks for your feedback, I am glad you like the overall idea. What value would it give to you as a student? @xIEUx

**Reply by Speb04** [reply]
I think having the assignments right next to a calendar tab can help students organise their schedules and slot out a certain day and time to do their assignments. Not having to swap between apps to do this can improve workflow and personally it would motivate me to do work as well as regularly reminding me about assignments and resubs I need to do. This combined with the Kanban board can really help students to focus and plan out time to study.

**Reply by WillTheDeveloper** [reply]
Yeah. Do you think having reminder emails or text messages would help you too?

**Reply by Speb04** [reply]
Definitely, I've turned in assignments late before because they have completely slipped my mind, I think if an assignment hasn't been handed in yet, a reminder 1 - 3 days before the due date would have helped me get work in on time.

**Reply by WillTheDeveloper** [reply]
I see. Yeah. I will look into implementing this! I do agree it would be super helpful for students.

Do you actually check your emails? I find that people don't check their emails often so do you think a text message would be better? Or would both serve you well?

**Reply by Speb04** [reply]
I dont think many students check emails often, I dont. Texts and if possible team messages would be the best options I think.

**Reply by WillTheDeveloper** [reply]
I see. I can imagine texts and emails working. Do you check teams a lot outside of college hours? Teams integration is an option.

**Reply by Speb04** [reply]
I dont but I know students do. The notifications on teams are what get me to open it, this is why I think texts are also a good option because it would be hard to ignore a notification from them whereas on email, I dont check notifications.

**Reply by WillTheDeveloper** [reply]
Okay -

Do you know when you refer to getting messages via teams, do you mean directly or in a teams team chat? Also, could you elaborate or clarify what makes you think that getting messages on teams is a good idea, I have done some research and it is definitely possible to do, just trying to think on a wider spectrum in regard to what it would achieve for people? If that makes sense? Like what is your thought process?

With that said, other people studying other subjects might use it a lot more but I am still curious about what you are thinking.

I think texts and emails would be best but I just want to challenge you a little but regarding the teams one but still something I will consider implementing.

**Reply by WillTheDeveloper** [reply]
@xIEUx this is your reply that you deleted -> Getting messaged directly through a chat would be the best way of getting to me personally and I imagine it is the same for most students because getting notifications/messages through a team is very common and could be messaging anybody, whereas if I am messaged directly it gets my attention and I will open it.

**Reply by Speb04** [reply]
Personally, when I get a notification from a team I know that that wasn't directly for me so I am less likely to check it, however if I know somebody is messaging me directly in a private chat, I make sure to always check them because they are directly to me. That's why I think a direct teams message would be the most effective way for me to be alerted, and I think it would be the most effective way to get most peoples attention.

**Reply by WillTheDeveloper** [reply]
Yeah, makes sense!
